### PR TITLE
[tests-only][full-ci] added test for mail notification in different languages when shared to group

### PR DESCRIPTION
**Pull request by ghost**
## Description
This PR adds the API test scenario for mail notification.
* Scenario: group members get an email notification in their respective languages when someone shares a folder with the group
* Scenario: group members get an email notification in their respective languages when someone shares a file with the group
* Scenario: group members get an email notification in their respective languages when someone shares a space with the group

## Related Issue
* Part of: https://github.com/owncloud/ocis/issues/6606

#### test environment:
* locally

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [X] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [X] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/apiNotification/emailNotification.feature [hunk 1]
```diff
@@ -70,3 +70,61 @@ Feature: Email notification
 
       Click here to view it: %base_url%/f/%space_id%
       """
+
+
+  Scenario: group members get an email notification in their respective languages when someone shares a folder with the group
+    And user "Carol" has been created with default attributes and without skeleton files
```
Comment (saw-jan) [inline tests/acceptance/features/apiNotification/emailNotification.feature:76]:
```suggestion
    Given user "Carol" has been created with default attributes and without skeleton files
```

File: tests/acceptance/features/apiNotification/emailNotification.feature [hunk 2]
```diff
@@ -70,3 +70,61 @@ Feature: Email notification
...
+      Hallo Carol King
+
+      %displayname% hat "HelloWorld" mit Ihnen geteilt.
+
+      Zum Ansehen hier klicken: %base_url%/files/shares/with-me
+      """
+
+
+  Scenario: group members get an email notification in their respective languages when someone shares a file with the group
+    And user "Carol" has been created with default attributes and without skeleton files
```
Comment (saw-jan) [inline tests/acceptance/features/apiNotification/emailNotification.feature:104]:
```suggestion
    Given user "Carol" has been created with default attributes and without skeleton files
```

File: tests/acceptance/features/apiNotification/emailNotification.feature [hunk 3]
```diff
@@ -70,3 +70,61 @@ Feature: Email notification
...
+    And user "Carol" should have received the following email from user "Alice"
+      """
+      Hallo Carol King
+
+      %displayname% hat "text.txt" mit Ihnen geteilt.
+
+      Zum Ansehen hier klicken: %base_url%/files/shares/with-me
+      """
+
+
```
Comment (saw-jan) [inline tests/acceptance/features/apiNotification/emailNotification.feature:130]: extra trailing new lines
```suggestion

```

**Review by saw-jan — APPROVED** [review]
LGTM :+1:

**Review by SwikritiT — APPROVED** [review]
LGTM

**Review by SagarGi — COMMENTED** [review]
LGTM :+1:

**Review by saw-jan — APPROVED** [review]
LGTM. But please state that you have added new scenarios or made new changes while re-requesting review because a new scenario was added after the PR was already approved.

**Comment by nirajacharya2** [comment]
@SagarGi @SwikritiT i added new new scenario to this PR please re-review it

**Review by SwikritiT — APPROVED** [review]

**Review by SagarGi — COMMENTED** [review]
LGTM :+1:

**Pull request merged**